### ISSUE TITLE
Merge Log Tool into MEASUR

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/AMO-Tools-Desktop#2895]
Once the Log Tool is complete, it will need to be merged into MEASUR

Log tool is the first step to Compressed air.  It has a few major functions
1- import logger data.  (despite what it says in the attached, probably just .csv files)
2 - assign column headers to data to make it so can work with the data
3 - Make graphs
4 - consolidate timestamp data into hourly data so user can define each day in the data as one of several  "day types" (e.g., 24 hour chunks where the air usage is high = "high day" or "week day"; 24 hr chunks where air usage is low = "weekends") Tool then takes all days of a given type and makes generic use profile for each {if tool could ID they data types itself, that would be more awesome)

[Logtool - Development overview.pptx](https://github.com/ORNL-AMO/AMO-Tools-Desktop/files/2978501/Logtool.-.Development.overview.pptx)

**Comment by koay9f** [comment]
This will need to be broken down into several issues once we are ready to start working on the Log Tool
